src/test/java/octo/waddle/hellodropwizard/resources/HelloWorldResourceTest.java

```diff
@@ -0,0 +1,24 @@
+package octo.waddle.hellodropwizard.resources;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.dropwizard.jackson.Jackson;
+import javax.ws.rs.core.Application;
+import octo.waddle.hellodropwizard.HelloDropwizardApplication;
+import octo.waddle.hellodropwizard.api.Message;
+import org.glassfish.jersey.test.JerseyTest;
+import org.junit.Test;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class HelloWorldResourceTest {
```

Comment:
Resource możesz testować na kilka sposobów,
Ten nie do końca jest ok, bo nie testujesz częsci Jerseyowej która mapuje ci requesta na parametry
Więc to że nie brakuje ci `name` w pathie nie zostało wykryte
Narazie dokończmy drobnicę z góry a potem zabierzemy się za to